### PR TITLE
Fix flakey tests with DurationUntilReset. Update docker example to V3 config.

### PR DESCRIPTION
**Pull request by ysawa0**
- Updated existing integration tests such that DurationUntilReset is not tested (guaranteed to be the same) due to flakiness
- Added new test for DurationUntilReset
- Update docker compose example to V3 API.
- Add example prom statsd exporter config.

### CODE REVIEW

File: test/integration/integration_test.go [hunk 1]
```diff
@@ -467,6 +466,21 @@ func testBasicBaseConfig(grpcPort, perSecond string, local_cache_size string) fu
...
+			context.Background(),
+			common.NewRateLimitRequest("another", [][][2]string{{{getCacheKey("key4", enable_local_cache), "durTest"}}}, 1))
+
+		time.Sleep(2 * time.Second) // Wait to allow duration to tick down
+
+		resp2, err := c.ShouldRateLimit(
+			context.Background(),
+			common.NewRateLimitRequest("another", [][][2]string{{{getCacheKey("key4", enable_local_cache), "durTest"}}}, 1))
+
+		if resp2.GetStatuses()[0].DurationUntilReset != nil {
```
Comment (mattklein123) [inline test/integration/integration_test.go:481]: Shouldn't we actually test that this isn't nil? When would it be nil?

**Review by mattklein123 — CHANGES_REQUESTED** [review]
Thanks a ton for fixing this. One small comment.

/wait

File: test/integration/integration_test.go (hunk 1 above)
Comment (ysawa0) [reply test/integration/integration_test.go:481]: I've been looking into this and there seems to be a bug where if local cache is on and a valid request like
`domain:"foo"  descriptors:{entries:{key:"hello"  value:"world"}}  hits_addend:1` is sent, the code thinks the key is empty and therefore returns a `overall_code:OK  statuses:{code:OK}` response with no duration until reset. Still digging into this, stay tuned...

Comment (ysawa0) [reply test/integration/integration_test.go]: Okay, fixed. Was just missing the corresponding local cache key in configs.

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Pull request merged**